triangle.php

```diff
@@ -1,111 +1,45 @@
-<html>
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="">
-    <link rel="stylesheet" href="css/tyle1.css">
-    <link rel="stylesheet" href="bootstrap-4.3.1/css/bootstrap.min.css">
-    <link rel="stylesheet" href="bootstrap-4.3.1/css/bootstrap.css">
-</head>
-<body>
-<nav class="navbar navbar-default navbar-expand-md navbar-light bg-light">
+<?php
+require 'vendor/autoload.php';
+
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+$twig = $container['twig'];
```

Comment:
@Legion112 This has an error,
What is the variable $container?

Reply:
https://github.com/HenintsoaHARINORO/ExoPHP/blob/f2041c6a0c399a5766c5b9c1073bbcd20e2f809d/messages.php#L8